<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerbooksajeckDialog</class>
 <widget class="QDialog" name="customerbooksajeckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 255, 0)</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_identitysajeck">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>180</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  book or booked</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gmailsajeck">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>  gmail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gmailsajeck">
   <property name="geometry">
    <rect>
     <x>144</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  Joey   @gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_identitysajeck">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>184</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>  identity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eidsajeck">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="text">
    <string>  eid</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eidsajeck">
   <property name="geometry">
    <rect>
     <x>146</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(85, 85, 127)</string>
   </property>
   <property name="placeholderText">
    <string>  ctg01</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updatecustomersajeck">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
